packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1001,6 +1029,12 @@ export const createTeamsCallAdapter: ({ userId, credential, locator, options }:
 // @beta
 export const createTeamsCallAdapterFromClient: (callClient: StatefulCallClient, callAgent: TeamsCallAgent, locator: CallAdapterLocator, options?: TeamsAdapterOptions | undefined) => Promise<TeamsCallAdapter>;
 
+// @public
+export interface CustomBackground {
```

Comment:
can we flatten this to 
```tsx
VideoBackgroundImage {
  id: string;
  url: string;
  name?: string;
}
```

Or do we need BackgroundImage to be nested in here?